prototypes/prototype.adoc

```diff
@@ -0,0 +1,18 @@
+:author: Han Solo <han@example.com>; Frodo Baggins <frodo@example.com>
+:state: predraft
+:revremark: State: {state}
+
+:showtitle:
+:toc: left
+:numbered:
+:icons: font
+
+////
+    This Source Code Form is subject to the terms of the Mozilla Public
+    License, v. 2.0. If a copy of the MPL was not distributed with this
+    file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+    Copyright 2017 <contribution>
+////
+
+# RFD <Number> <Title>
```

Comment:
Do we want "# ..." for the h1 or "= ..." as the AsciiDoc user-guide seems to suggest? (Note that 'make check' and the bot tooling don't yet recognize "= ...".)